bundles/GraphicServiceOrder/Form/GraphicServiceOrderForm.php

```diff
@@ -63,8 +64,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // Add file upload endpoint.
         $helper = $this->container->get('oneup_uploader.templating.uploader_helper');
         $endpoint = $helper->endpoint('gsorder');
-
         $builder
+            ->add('full_name', TextType::class, [
+                'label' => 'service_order_form.full_name.label',
+                'constraints' => [
+                    new NotNull(['groups' => 'base']),
+                ],
+                'attr' => ['class' => 'form-control'],
```

Comment:
All styling should be handled by the/a form theme (https://symfony.com/doc/current/form/form_customization.html#form-themes).

Reply:
Hmm, all classes, were somehow duplicate, we seem to have implemented some kind of bootstrap form theme that included the clases by default. So just removed a lot of attributes from form class.